test/lib/cli/commands/base.js

```diff
@@ -1,21 +1,31 @@
 const test = require("ava");
 const path = require("path");
 const execa = require("execa");
+const sinon = require("sinon");
 const pkg = require("../../../../package.json");
 const ui5Cli = path.join(__dirname, "..", "..", "..", "..", "bin", "ui5.js");
 const ui5 = (args, options = {}) => execa(ui5Cli, args, options);
 
-test("ui5 --version", async (t) => {
+test.beforeEach((t) => {
+	t.context.consoleLogStub = sinon.stub(console, "log");
+	t.context.processExitStub = sinon.stub(process, "exit");
```

Comment:
See https://github.com/SAP/ui5-cli/blob/master/test/lib/cli/commands/init.js#L36

Reply:
Ok, calling restore in afterEach.always should not cause these problems 👍